### PR TITLE
Feature/52

### PR DESCRIPTION
**Pull request by maheshrayas**

### CODE REVIEW

**Comment by maheshrayas** [comment]
Attaching preview of the changes :
 
Screen Shot 2019-05-11 at 6 38 47 pm
Screen Shot 2019-05-11 at 6 39 00 pm

File: pkg/up/util.go [hunk 1]
```diff
@@ -241,3 +245,18 @@ func contains(arr []string, str string) bool {
 	}
 	return false
 }
+
+func getMaxLenOfServiceName(podsName []string) int {
+	max := len(podsName[0])
```
[the remark this answers is not in the document]
Comment (jbrekelmans) [reply pkg/up/util.go]: This assumes there is at least 1 service, which need not be the case

[the remark this answers is not in the document]
Comment (jbrekelmans) [reply pkg/up/util.go]: Should rename `podsName` to `serviceNames`

[the remark this answers is not in the document]
Comment (maheshrayas) [reply pkg/up/util.go]: Fixed

File: pkg/up/up.go [hunk 2]
```diff
@@ -620,6 +620,7 @@ func (u *upRunner) updateAppMaxObservedPodStatus(pod *v1.Pod) error {
 				completedChannel := make(chan interface{})
 				u.completedChannels = append(u.completedChannels, completedChannel)
 				go func() {
+					color := getRandomColor()
```
[the remark this answers is not in the document]
Comment (maheshrayas) [reply pkg/up/up.go]: Fixed

File: pkg/up/util.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (maheshrayas) [reply pkg/up/util.go]: PTAL @jbrekelmans

**Pull request merged**